deploy/scripts/src/authentication/test.ts

```diff
@@ -203,14 +211,20 @@ export function signUp(): void {
   const mfaOption: mfaType = Math.random() <= 0.5 ? 'SMS' : 'AUTH_APP'
   iterationsStarted.add(1)
 
-  // B01_SignUp_01_OrchStubSubmit
-  res = orchStubSubmit(groups)
+  // B01_SignUp_01_StubSubmit based on the route i.e. RP or ORCH
+  if (route === 'ORCH') {
+    res = orchStubSubmit(groups)
+  } else if (route === 'RP') {
+    res = rpStubSubmit(groups)
+  } else {
+    throw new Error(`Invalid route: ${route}`) // Handle invalid route values
```

Comment:
Don't think this is required as we are already validating the value of the ROUTE earlier.

Reply:
Agree, removed entire if else block